src/CommunityToolkit/Xamarin.CommunityToolkit.UnitTests/Converters/IsNullOrEmptyConverter_Tests.cs

```diff
@@ -1,15 +1,25 @@
-﻿using System.Globalization;
-using Xamarin.CommunityToolkit.Converters;
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
 using NUnit.Framework;
+using Xamarin.CommunityToolkit.Converters;
 
 namespace Xamarin.CommunityToolkit.UnitTests.Converters
 {
 	public class IsNullOrEmptyConverter_Tests
 	{
-		[TestCase(null, true)]
-		[TestCase("", true)]
-		[TestCase("Test", false)]
-		[TestCase(typeof(IsNullOrEmptyConverter), false)]
+		public static IEnumerable<object?[]> GetData() => new List<object?[]>
+		{
+			new object?[] { null, true },
+			new object[] { string.Empty, true },
+			new object[] { "Test", false },
+			new object[] { typeof(IsNullOrEmptyConverter), false },
+			new object[] { new List<string>(), true },
+			new object[] { new List<string>() { "TestValue" }, false },
+			new object[] { Enumerable.Range(1, 3), false },
+		};
```

Comment:
Add test for whitespace string:
`new object[] { " ", true },`

Reply:
Done!